nemo/collections/nlp/data/text_normalization/decoder_dataset.py

```diff
@@ -220,6 +282,64 @@ def __getitem__(self, idx):
     def __len__(self):
         return len(self.examples)
 
+    def batchify(self, batch_size: int):
+        """
+        Creates a batch
```

Comment:
can you use DataCollatorForSeq2Seq here too, and pass model as argument?
And by creating Dataloader to help you generate the batch keys needed? Since these could change with different model?